### PR TITLE
release: 2.60.4

### PR DESCRIPTION
**Pull request by mvo5**
Changelog for 2.60.4

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-# New in snapd 2.60.3:
+# New in snapd 2.60.4:
+* Switch to plug/slot in the "qualcomm-ipc-router" interface
+* Fix "customer-device" udev KERNEL values
```
Comment (olivercalder) [inline NEWS.md:3]: s/customer/custom/

**Review by olivercalder — CHANGES_REQUESTED** [review]
Tiny typo, otherwise looks great!

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
-# New in snapd 2.60.3:
+# New in snapd 2.60.4:
+* Switch to plug/slot in the "qualcomm-ipc-router" interface
```
Comment (pedronis) [inline NEWS.md:2]: maybe: ... keeping also backward compatibility

**Review by pedronis — APPROVED** [review]
looks good apart typo

**Review by olivercalder — APPROVED** [review]
LGTM

**Pull request merged**